packages/ckeditor5-table/src/tablecellproperties.js

```diff
@@ -35,7 +36,7 @@ export default class TableCellProperties extends Plugin {
 	 * @inheritDoc
 	 */
 	static get requires() {
-		return [ TableCellPropertiesEditing, TableCellPropertiesUI ];
+		return [ TableCellPropertiesEditing, TableCellPropertiesUI, TableCellWidthEditing ];
```

Comment:
The `TableCellWidthEditing` plugin should be required by `TableCellPropertiesEditing` rather than the glue plugin.

Reason for that is that previously it was the `TableCellPropertiesEditing` plugin that implemented conversion. However, now when you moved it to the "glue plugin", the `TableCellPropertiesEditing` doesn't do that anymore - which is a breaking change 🙂